packages/account/src/test-utils/launchNode.test.ts

```diff
@@ -38,14 +38,36 @@ describe('launchNode', () => {
   });
 
   it('cleanup kills the started node', async () => {
-    const { cleanup, url } = await launchNode();
+    const killSpy = vi.spyOn(process, 'kill');
+
+    const { cleanup, url, pid } = await launchNode();
     expect(await fetch(url)).toBeTruthy();
 
     cleanup();
 
+    expect(killSpy).toHaveBeenCalledTimes(1);
+    // adding a minus prefix kills the process group
+    expect(killSpy).toHaveBeenCalledWith(-pid);
+
     await waitUntilUnreachable(url);
   });
 
+  /**
+   * Spawning the child process in a detached state
+   * Results in the OS assigning a process group to the child.
+   * Combining that with `process.kill(-pid)`,
+   * which sends a "kill process group" signal to the OS,
+   * ensures that the node will be killed.
+   */
+  it('spawns the fuel-core node in a detached state', async () => {
```

Comment:
I wonder if both tests shouldn't be just one, mocking both kill and spawn methods simultaneously and ensuring both are called when as expected.

Reply:
You are right. https://github.com/FuelLabs/fuels-ts/pull/2759/commits/aa4f52453974d0172bd39f74d76af37e91e0c126